contrib/completion/bash/docker

```diff
@@ -2643,6 +2643,9 @@ _docker_service_update() {
 	if [ "$subcommand" = "create" ] ; then
 		options_with_args="$options_with_args
 			--container-label
+			--dns
+			--dns-option
```

Comment:
@sdurrheimer Seems to be correct, though:
```bash
$ docker service create --dns-option
flag needs an argument: --dns-option
See 'docker service create --help'.
$ docker service create --dns-options
unknown flag: --dns-options
See 'docker service create --help'.
```
Maybe this was changed in the meantime, see also #28369